sdk/core/test/unit/services/WalletStorageService.test.ts

```diff
@@ -34,7 +34,7 @@ let encryptedSeed: string
 
 const region = 'eu-west-2'
 
-const createWalletStorageService = () => {
+const createWalletStorageService = (queryBloomVault?: boolean) => {
```

Comment:
BTW why it doesn't have default option? 
`const createWalletStorageService = (queryBloomVault: boolean = true) => {`
it could then simplify this block of code
`queryBloomVault: queryBloomVault === false ? false : true,`
to 
`queryBloomVault`